launcher/src/main/java/org/apache/spark/launcher/JavaModuleOptions.java

```diff
@@ -40,7 +40,8 @@ public class JavaModuleOptions {
       "--add-opens=java.base/sun.nio.cs=ALL-UNNAMED",
       "--add-opens=java.base/sun.security.action=ALL-UNNAMED",
       "--add-opens=java.base/sun.util.calendar=ALL-UNNAMED",
-      "--add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED"};
+      "--add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED",
+      "-Djdk.reflect.useDirectMethodHandle=false"};
```

Comment:
It is better to rename `JavaModuleOptions` as `JavaExtraOptions`. Do you suggest completing the corresponding refactor work in this pr? @dongjoon-hyun

Reply:
My idea is that this pr solves the issue first if need, and then do the rename work by a separate pr

Reply:
Ya, I agree with you. Let's postpone the renaming.

Reply:
Thanks @dongjoon-hyun